core/src/replay_stage.rs

```diff
@@ -2083,6 +2100,7 @@ impl ReplayStage {
             );
 
             if !Self::check_propagation_for_start_leader(poh_slot, parent_slot, progress_map) {
+                // We checked that parent_slot hasn't been dumped, so we should get it in the progress map.
```

Comment:
I think it's fine to not add this comment since the `expect` below explains the situation.

Reply:
Done.